tests/test_compound.py

```diff
@@ -42,9 +42,9 @@ def test_json_schema(self):
 
     def _test_schema(self, schema):
         tw = TextWiser(Embedding.Compound(schema=schema), dtype=torch.float32)
-        expected = torch.tensor([[-1.5983865261,  1.8820908070,  0.1802073568],
-                                 [-1.8616025448, -0.4420224428, -0.9159017205],
-                                 [-2.0401582718, -1.0712100267,  0.6945561171]], dtype=torch.float32)
+        expected = torch.tensor([[1.5983865261, -1.8820908070,  -0.1802073568],
```

Comment:
I understand the test changes. Shall we expect/account for such behavior change "internally" where textwiser is used?

Reply:
Only for pipelines that rely on SVD & UMAP (& possibly Doc2Vec). That being said, before this PR those models would be non-deterministic with latest requirements. So some of the numbers could change after this PR, but if you installed TextWiser before this PR to a different machine, there was a possibility that you wouldn't get the same numbers either.

I think we should be on the lookout for any tests failing, but I don't think we're relying on the exact numbers generated by SVD or UMAP anywhere in the rest of our open source stack.